scripts/v.in.wfs/v.in.wfs.html

```diff
@@ -13,6 +13,9 @@ <h3>WFS import without credentials</h3>
 # run in Latitude-Longitude location (EPGS code 4326):
 # download "sentinel:mgrs" layer:
 v.in.wfs url="https://geoserver.mundialis.de/geoserver/sentinel/wfs?" name="sentinel:mgrs" output=sentinel2_mgrs
+# download  NRW ALKIS "ave:Flurstueck" attribute:
+v.in.wfs url="https://www.wfs.nrw.de/geobasis/wfs_nw_alkis_vereinfacht?" output=wfs_alkis_vereinfacht srs=25832
+name="ave:Flurstueck" version="2.0.0" aoi="335719,5615930,354798,5636274" layer="Flurstueck"
```

Comment:
Change the example because of the removed `aoi` paramter.
E.g. you can add `g.region` before and then use `-r`.